plugin-server/src/ingestion/ingestion-consumer.ts

```diff
@@ -27,6 +25,12 @@ import { MemoryRateLimiter } from './utils/overflow-detector'
 // Must require as `tsc` strips unused `import` statements and just requiring this seems to init some globals
 require('@sentry/tracing')
 
+const ingestionPartitionKeyOverflowed = new Counter({
+    name: 'ingestion_partition_key_overflowed',
+    help: 'Indicates that a given key has overflowed capacity and been redirected to a different topic. Value incremented once a minute.',
```

Comment:
nit: It's not really incremented once a minute, right?

Reply:
just copied and pasted from elsewhere. (btw in the future feel free to propose a commit suggestion for this stuff (you've done the thinking to get to this point so may as well propose your improvement directly 👍 )

Reply:
Renamed it and removed the partition_key as it was too high cardinality anyways